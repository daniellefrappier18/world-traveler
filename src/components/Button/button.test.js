import React from "react";
import { Button } from "./button.component";
import renderer from "react-test-renderer";
import { cleanup } from "@testing-library/react";

describe("Button component", () => {
  // test component with props
  it("Button renders correctly all Props", () => {
    const tree = renderer
      .create(
        <Button
            ariaLabel="test"
            href="https://www.google.com"
            target="_blank"
            text="test"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  afterEach(cleanup);
});