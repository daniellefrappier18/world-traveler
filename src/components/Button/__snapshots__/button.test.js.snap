// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Button renders correctly all Props 1`] = `
<button
  className="button"
>
  <a
    aria-label="test"
    href="https://www.google.com"
    target="_blank"
  >
    test
  </a>
</button>
`;
